CONTRIBUTING.md

```diff
@@ -32,6 +32,12 @@ reported the issue. Please try to include as much information as you can. Detail
 
 Contributions via pull requests are much appreciated.
 
+### TL;DR;
```

Comment:
```suggestion
### Summary
```

TLDR gives me the feeling that I don't have to read this. But it's still important to read through at some point.